jwala-webapp/src/main/webapp/resources/js/react/group-operations/GroupOperationsDataTable.js

```diff
@@ -843,12 +839,8 @@ var GroupOperationsDataTable = React.createClass({
     jvmConfirmDeleteCallback: function(data) {
         let self = this;
         jvmService.deleteJvm(data.id.id, true).then(function(e){
-            // Remove deleted row
-            $("[id*='jvm-child-table']").find("td:contains('" + data.jvmName + "')").filter(function() {
-                if ($(this).text() === data.jvmName) {
-                    $(this).parent().remove();
-                }
-            });
+            $.alert("JVM " + data.jvmName + " was successfully deleted. Jwala will need to refresh to display the latest data and recompute the states.",
```

Comment:
I can't just refresh the JVMs since the JVM is attached to the group data unlike Web Server. I wanted to make JVMs load like Web Server before but the team decided not to then. I can convert it to the way Web Server does it but it's going to take a lot of work - so might as well refactor and write it the right way this time. I tried doing it just to gauge the time it will take, surprisingly I was able to refactor the group table in 3 hours or less and the code looks great. I didn't know why that one escaped me before - probably due to the learning curve.